autoscale/src/main/resources/schema/app/20200318083430_CB-5734_cluster_table_enhancements.sql

```diff
@@ -5,8 +5,6 @@ ALTER TABLE cluster ADD COLUMN IF NOT EXISTS cb_stack_name VARCHAR(255);
 
 ALTER TABLE cluster ADD COLUMN IF NOT EXISTS cb_stack_type VARCHAR(255);
 
-ALTER TABLE cluster ADD COLUMN IF NOT EXISTS autoscale_mode VARCHAR(255);
```

Comment:
This essentially means we are back to inefficient queries (full table scans), correct?

Reply:
I have to investigate how to optimize this but I think we can avoid full table scans.